plugins/modules/ip_assign.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""This module allows users to assign IP addresses to multiple Linodes in one Region."""
+
+from __future__ import absolute_import, division, print_function
+
+from typing import Any, Optional
+
+import ansible_collections.linode.cloud.plugins.module_utils.doc_fragments.ip_assign as docs
+from ansible_collections.linode.cloud.plugins.module_utils.linode_common import (
+    LinodeModuleBase,
+)
+from ansible_collections.linode.cloud.plugins.module_utils.linode_docs import (
+    global_authors,
+    global_requirements,
+)
+from ansible_collections.linode.cloud.plugins.module_utils.linode_helper import (
+    filter_null_values,
+)
+from ansible_specdoc.objects import FieldType, SpecDocMeta, SpecField
+
+linode_ip_assignments_spec: dict = {
+    "address": SpecField(
+        type=FieldType.string,
+        required=True,
+        description=["The IPv4 address or IPv6 range."],
+    ),
+    "linode_id": SpecField(
+        type=FieldType.integer,
+        required=True,
+        description=["ID of the Linode."],
+    ),
+}
+
+spec = {
+    # Disable the default values
+    "state": SpecField(type=FieldType.string, required=False, doc_hide=True),
+    "label": SpecField(type=FieldType.string, required=False, doc_hide=True),
+    "assignments": SpecField(
+        type=FieldType.list,
+        element_type=FieldType.dict,
+        suboptions=linode_ip_assignments_spec,
+        required=True,
+        editable=True,
+        description=["List of assignments to make."],
+    ),
+    "region": SpecField(
+        type=FieldType.string,
+        required=True,
+        description=["The Region to operate in."],
+    ),
+}
+
+SPECDOC_META = SpecDocMeta(
+    description=["Assign IPs to Linodes in a given Region."],
+    requirements=global_requirements,
+    author=global_authors,
+    options=spec,
+    examples=docs.specdoc_examples,
+    return_values={},
+)
+
+
+class Module(LinodeModuleBase):
+    """Module for assigning IPs to Linodes in a given Region"""
+
+    def __init__(self) -> None:
+        self.module_arg_spec = SPECDOC_META.ansible_spec
+        self.results = {"changed": False}
+
+        super().__init__(
+            module_arg_spec=self.module_arg_spec,
+            required_one_of=[],
+            mutually_exclusive=[],
```

Comment:
If we don't need to specify `required_one_of` and `mutually_exclusive` in a module, is it okay to just remove them here or we have to pass in the empty array?